python/kserve/test/test_server.py

```diff
@@ -240,14 +243,14 @@ def test_list_models_v1(self, http_server_client):
 
     def test_predict_v1(self, http_server_client):
         resp = http_server_client.post('/v1/models/TestModel:predict',
-                                       data=b'{"instances":[[1,2]]}')
+                                       content=b'{"instances":[[1,2]]}')
```

Comment:
To fix the warning
DeprecationWarning: Use 'content=<...>' to upload raw bytes/text content.